teams/migration/README.md

```diff
@@ -0,0 +1,15 @@
+# Migration Team
+
+This team works for migration, replication, synchronization and transmission of data into and out of the TiDB ecosystem.
+
+## Members
+
+See [membership.json](membership.json)
+
+## Repositories
+
+* [ticdc](https://github.com/pingcap/ticdc)
+* [dm](https://github.com/pingcap/dm)
+* [dumpling](https://github.com/pingcap/dumpling)
+* [tidb-binlog](https://github.com/pingcap/tidb-binlog)
+* [br](https://github.com/pingcap/br)
```

Comment:
since BR v5.2 is moved into `pingcap/tidb` while BR v5.1 remains at `pingcap/br`, it is better `pingcap/br` be managed by `team-tidb` too.

Reply:
Fine. The border is a bit vague.

Reply:
Updated.